Parkinson/TasksAndSteps/FilenameTranslation.json

```diff
@@ -0,0 +1,11 @@
+{
+    "audio_countdown" : "audio_countdown.m4a",
+    "audio_audio" : "audio_audio.m4a",
+    "accelerometer_tapping" : "accelerometer_tapping",
+    "pedometer_walking.outbound" : "pedometer_walking.outbound",
+    "accelerometer_walking.outbound" : "accel_walking_outbound.json",
+    "deviceMotion_walking.outbound" : "deviceMotion_walking_outbound.json",
+    "accelerometer_walking.rest" : "accelerometer_walking.rest",
+    "deviceMotion_walking.rest" : "deviceMotion_walking.rest",
+    "momentInDayFormat" : "momentInDayFormat.json"
+}
```

Comment:
The long version is here: https://github.com/ResearchKit/ResearchKit/issues/323
The short version is the Stable branch of RK no longer supports having a resultIdentifier and stepIdentifier in the [ORKFileResult fileURL]. We had a hack in APCDataArchiver to get a filename from the URL. So this commit broke that functionality. Therefore, we could no longer guarantee you the filenames you need. This filename translation file takes the resultIdentifier and stepIdentifier and concatenates and translates the string into your filename.